src/main/java/org/broadinstitute/consent/http/db/ElectionDAO.java

```diff
@@ -356,34 +484,34 @@ void updateElectionById(@Bind("electionId") Integer electionId,
      * @return Dac for this election
      */
     @UseRowMapper(DacMapper.class)
-    @SqlQuery("select d0.* from ( " +
-            "   select d1.*, e1.electionId from dac d1 " +
-            "     inner join dataset ds1 on d1.dac_id = ds1.dac_id " +
-            "     inner join election e1 on e1.datasetId = ds1.dataset_id and e1.electionId = :electionId " +
-            " union " +
-            "   select d2.*, e2.electionId from dac d2 " +
-            "     inner join dataset ds2 on d2.dac_id = ds2.dac_id " +
-            "     inner join consent_associations a2 on a2.dataset_id = ds2.dataset_id " +
-            "     inner join election e2 on e2.referenceId = a2.consent_id and e2.electionId = :electionId " +
+    @SqlQuery("SELECT d0.* FROM ( " +
+            "   SELECT d1.*, e1.election_id FROM dac d1 " +
+            "     INNER JOIN dataset ds1 on d1.dac_id = ds1.dac_id " +
+            "     INNER JOIN election e1 ON e1.dataset_id = ds1.dataset_id AND e1.election_id = :electionId " +
+            " UNION " +
+            "   select d2.*, e2.election_id FROM dac d2 " +
+            "     INNER JOIN dataset ds2 ON d2.dac_id = ds2.dac_id " +
+            "     INNER JOIN consent_associations a2 ON a2.dataset_id = ds2.dataset_id " +
+            "     INNER JOIN election e2 ON e2.reference_id = a2.consent_id AND e2.election_id = :electionId " +
             " ) as d0 limit 1 ") // `select * from (...) limit 1` syntax is an hsqldb limitation
```

Comment:
Observation: This query can be re-written, but not for this PR. When we were using mysql, we tested against hsqldb. Since the postgres migration and true db integration tests, we don't need to contort our queries any longer to work in both mysql+hsqldb formats.